apps/docs/pages/guides/functions/auth.mdx

```diff
@@ -4,60 +4,91 @@ export const meta = {
   id: 'auth',
   title: 'Integrating With Supabase Auth',
   description: 'Supabase Edge Functions and Auth.',
+  subtitle: 'Supabase Edge Functions and Auth.',
 }
 
-Edge Functions work seamlessly with [Supabase Auth](/docs/guides/auth), allowing you to identify which user called your function.
+Edge Functions work seamlessly with [Supabase Auth](/docs/guides/auth).
 
-When [invoking](/docs/reference/javascript/invoke) a function with one of the [client libraries](/docs/reference), the logged in user's JWT is automatically attached to the function call and becomes accessible within your function.
+## Auth Context
 
-This is important, for example, to identify which customer's credit card should be charged. You can see this concept end-to-end in our [Stripe example app](https://github.com/supabase-community/expo-stripe-payments-with-supabase-functions).
+When a user makes a request to an Edge Function, we can use the Authorization header to set the Auth context in the Supabase client:
 
-## Auth Context & RLS
+```js mark=10
+import { serve } from 'https://deno.land/std@0.177.0/http/server.ts'
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'
+
+serve(async (req: Request) => {
+
+  const authHeader = req.headers.get('Authorization')!
+  const supabaseClient = createClient(
+    Deno.env.get('SUPABASE_URL') ?? '',
+    Deno.env.get('SUPABASE_ANON_KEY') ?? '',
+    { global: { headers: { Authorization: authHeader } } }
+  )
+
+})
+```
+
+Importantly, this is done _inside_ the `serve()` function, so that the Authorization header is set for each request.
 
-By creating a supabase client with the auth context from the function, you can do two things:
+## Fetching the user
 
-1. Get the user object.
-2. Run queries in the context of the user with [Row Level Security (RLS)](/docs/guides/auth/row-level-security) policies enforced.
+After initializing a Supabase client with the Auth context, we can use `getUser()` to fetch the user object, and run queries in the context of the user with [Row Level Security (RLS)](/docs/guides/auth/row-level-security) policies enforced.
 
-```js mark=14,17:19,22:23 supabase/functions/select-from-table-with-auth-rls/index.ts
+```js mark=13:14
 import { serve } from 'https://deno.land/std@0.177.0/http/server.ts'
 import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'
 
 serve(async (req: Request) => {
```

Comment:
```suggestion
import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'

Deno.serve(async (req: Request) => {
```

Not part of this PR I know, but need to fix it at the same time if we're changing above